src/sentry/grouping/ingest/seer.py

```diff
@@ -200,8 +200,21 @@ def _circuit_breaker_broken(event: Event, project: Project) -> bool:
 
 
 def _has_empty_stacktrace_string(event: Event, variants: dict[str, BaseVariant]) -> bool:
+    # For purposes of a temporary debug log - will be removed as soon as the mysterious behavior
+    # is sorted
+    logger_extra = {
+        "event_id": event.event_id,
+        "project_id": event.project_id,
+        "platform": event.platform,
+        "has_too_many_frames_result": has_too_many_contributing_frames(
```

Comment:
Just checking that this isn't expensive to run here? I assume this log happens only on group creation

Reply:
It runs on every event, but no, it's not expensive - all it does is check a few different pre-computed values (and collect metrics, if `record_metrics` is true, but in this case  it's false, so that's a moot point).

https://github.com/getsentry/sentry/blob/520f8d9ea05fd71061b5ed9b0c28da460a7144eb/src/sentry/seer/similarity/utils.py#L323-L382

If you still think it's an issue, I can move the call to where the log is, which would drastically cut down the number of times that it runs. I originally had it that way, but in the end, especially since it's hopefully only going to be in place for a day or two while I debug this, it just seemed easier to pass one metadata blob rather than all the stuff needed to compute said blob's values. LMK if you want me to change it.

Reply:
I think that as long as it won't cause a performance issue in prod I'm fine with it as is